### ISSUE TITLE
如何获取与每个分类的相关度得分

### ISSUE THREAD

**Issue by lijc210** [gaussic/text-classification-cnn-rnn#56]
作者你好，predict.py默认只能打印出最相关的一个分类，怎么打印出分类的得分？

**Comment by gaussic** [comment]
阅读这个项目需要对 TensorFlow 有一定的认识，获取分数只需要修改模型的输出部分就行了。

**Comment by hao1032** [comment]
@lijc210
将 predict.py 里面的
`y_pred_cls = self.session.run(self.model.y_pred_cls, feed_dict=feed_dict)
        return self.categories[y_pred_cls[0]]`
修改为
` y_pred_cls = self.session.run(tf.nn.softmax(self.model.logits), feed_dict=feed_dict)
        for index, item in enumerate(y_pred_cls[0]):
            print("{}, {:.5%}".format(self.categories[index], item))
        # return self.categories[y_pred_cls[0]]`

可以获取每个分类的概率：
体育, 0.00153%
财经, 0.03829%
房产, 0.00058%
家居, 0.00674%
教育, 0.24815%
科技, 99.41654%
时尚, 0.01558%
时政, 0.22205%
游戏, 0.04007%
娱乐, 0.01046%
None
体育, 99.99326%
财经, 0.00021%
房产, 0.00444%
家居, 0.00000%
教育, 0.00084%
科技, 0.00001%
时尚, 0.00001%
时政, 0.00018%
游戏, 0.00013%
娱乐, 0.00092%
None
